content/en/users/tutorials/data-transfer-grid-storage/_index.md

```diff
@@ -0,0 +1,229 @@
+FINISHED
+```
+
+which specify the server that control the transfer, the source and the transfer
+itself. The last option '-o' specify that the file should be overwritten if
+present on the destination. If this option is not present and a file with the
+same name exists on the destination, the transfer itself will fail. If you use
+this option you should make sure that it is safe to do so.
+
+## Using the WebFTS Data Transfer interface
+
+### Access the WebFTS interface
+
+The WebFTS is accessible at this URL, [WebFTS](https://webfts.cern.ch/)
+Similarly to what has been done from the command line interface you need to
+provide our private key for delegation of the credential. To do that you use the
+following command:
+
+```shell
+$ openssl pkcs12 -in yourCert.p12 -nocerts -nodes | openssl rsa
+Enter Import Password:
+writing RSA key
+-----BEGIN RSA PRIVATE KEY-----
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+...
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+-----END RSA PRIVATE KEY-----
+```
+
+Which extract the private key in RSA format and you can paste it in the windows
+that opens:
+
+![WebFTS credential delegation](WebFTS-credential_delegation.png)
+
+And select the desired VO. Once the delegation is set it's possible to move to
+the following steps.
+
+**Warning**: please be careful and avoid sharing this information with any third
+party or saving this information in plain text. WebFTS uses the key to acquire a
+proxy certificate on your behalf as described previously and doesn't store it.
+
+### Submitting a transfer
+
+The tab `Submit a transfer` is divided in two parts in which is possible to add
+two endpoints that can be used both as source or destination. After adding the
+URL for the two endpoints, it is possible to browse and select the files and
+directories to be transferred. In the destination select the destination
+directory. In the following example the file 1MB has been selected and by simply
+clicking the arrow in the middle directing to the right you are instructing the
+system to copy the file from the storage and path on the left to the one on the
+right:
+
+![WebFTS submit transfer](WebFTS-submit_transfer.png)
+
+Similarly to what seen on the comman line interface, we have the option to overwrite the destination if it already exists. To enable this option tick the 'Overwrite Files' below the arrow for the transfer.
+On the top of the page is also shown a confirmation that the transfer has been
+submitted successfully. This same webpage shows the status of the current
+transfers in the `My jobs` tab shown in the following screeshot:
+
+![WebFTS my jobs](WebFTS-my_jobs.png)
+
+Each line on the list shows a different job. if clicked it will expand showing some additional details. Of particular interest can be the reason for the failure of a job which can shown by moving the mouse pointer over the 'File ID' on the detailed view.
```

Comment:
```suggestion
Each line on the list shows a different job. When clicked it will expand to show
additional details. The reason for the failure of a job which can be seen by
moving the mouse pointer over the `File ID` on the detailed view.
```